src/hotspot/share/opto/memnode.cpp

```diff
@@ -1716,7 +1716,7 @@ Node* LoadNode::split_through_phi(PhaseGVN* phase) {
 AllocateNode* LoadNode::is_new_object_mark_load(PhaseGVN *phase) const {
```

Comment:
You can also remove the `phase` parameter of this method since there is now no usage left.